docs/source/performance.md

```diff
@@ -0,0 +1,96 @@
+
+## Node tuning
+
+Starting from Operator 1.6, a new CRD called NodeConfig is available, allowing users to target Nodes which should be tuned.
+When a Node is supposed to be optimized, the Scylla Operator creates a DaemonSet covering these Nodes.
+Nodes matching the provided placement conditions will be subject to tuning.
+
+Below example NodeConfig tunes nodes having `scylla.scylladb.com/node-type=scylla` label:
+```
+apiVersion: scylla.scylladb.com/v1alpha1
+kind: NodeConfig
+metadata:
+ name: cluster
+spec:
+ placement:
+   nodeSelector:
+     scylla.scylladb.com/node-type: scylla
+```
+For more details about new CRD use:
+```
+kubectl explain nodeconfigs.scylla.scylladb.com/v1alpha1
+```
+
+For all optimizations we use a Python script available in the Scylla image called perftune. 
+Perftune - contains performance optimization - it’s able to tune the kernel, network and disk devices, spread IRQs across CPUs and more.
+
+Tuning consists of two separate optimizations: common node tuning, and tuning based on Scylla Pods and their resource assignment.
+Node tuning is executed when tuning Pod lands on Node. Pod tuning is executed when Scylla Pod lands on the same Node.
+
+Scylla does not like sharing, it works most efficiently when it’s not interrupted. 
+One of the most common causes of context-switching are network interrupts. Packets coming to a node need to be processed, 
+and this requires CPU shares.  
+
+On K8s we always have at least a couple of processes running on the node: kubelet, kubernetes provider applications, daemons etc. 
+These processes require CPU shares, so we cannot dedicate entire node processing power to Scylla, we need to leave space for others.  
+We take advantage of it, and we pin IRQs to CPUs not used by any Scylla Pods exclusively.
+
+Tuning resources are created in a special namespace called `scylla-operator-node-tuning`.
+
+The tuning is applied only to pods with `Guaranteed` QoS class. Please double check your ScyllaCluster resource specification
+to see if it meets all conditions.
+
+## Kubernetes tuning
+
+By default, the kubelet uses the CFS quota to enforce pod CPU limits.  
+When the node runs many CPU-bound pods, the workload can move around different CPU cores depending on whether the pod 
+is throttled and which CPU cores are available.
+However, kubelet may be configured to assign CPUs exclusively, by setting the CPU manager policy to static.
+
+Setting up kubelet configuration is provider specific. Please check the docs for your distribution or talk to your
+provider.
+
+Pods which receive a Guaranteed Quality of Service class can take advantage of this option. 
+As Guaranteed pods whose containers fit the requirements for being statically assigned are scheduled to the node, 
+CPUs are removed from the shared pool and placed in the cpuset for the container.
+
+In our case there are two requirements each ScyllaCluster must fulfill to receive a Guaranteed QoS class:
```

Comment:
would it be better to adjust the working to match https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed ?

Reply:
I would like to leave explicit mention about agentResources, because users are not aware of it